### ISSUE TITLE
Annotations with Group/Host special characters on Grafana 2.6

### ISSUE THREAD

**Issue by poxin13** [alexanderzobnin/grafana-zabbix#134]
Sorting by Group or Host in annotations if their names have special characters will not return values.

So far this involves spaces, dots, and hyphens.

```
jsonrpc: "2.0", method: "trigger.get",…}
auth: "1bf171262666d9d7c499241ff4303eeb"
id: 1
jsonrpc: "2.0"
method: "trigger.get"
params: {output: ["triggerid", "description"], search: {description: ""}, searchWildcardsEnabled: true,…}
expandDescription: true
host: "ticket.mydomain.com"
output: ["triggerid", "description"]
0: "triggerid"
1: "description"
search: {description: ""}
description: ""
searchWildcardsEnabled: true
```
Response: {"jsonrpc":"2.0","result":[],"id":1}

**Comment by poxin13** [comment]
Likely related to #116

**Comment by alexanderzobnin** [comment]
Fixed by 4bcf741d66aeb51f7576df87ab630cb6af17c2bd
